presto-native-execution/presto_cpp/main/common/Configs.h

```diff
@@ -116,6 +116,10 @@ class SystemConfig : public ConfigBase {
       "http-server.enable-access-log"};
   static constexpr std::string_view kHttpEnableStatFilter{
       "http-server.enable-stats-filter"};
+  /// The options to configure the max quantized memory allocation size to store
+  /// the received http response data.
```

Comment:
I think we need a more detailed explanation on this parameter, preferrably with example to explain. This is very hard to understand.